src/includes/tip-in-list.njk

```diff
@@ -1,7 +1,8 @@
 {% set nbSupportedBrowsers %}{{ post.data.tags | onlyBrowsers | length }}{% endset %}
 <li class="hover-box tip {{ "unique-browser-feature" if nbSupportedBrowsers == 1 }}">
   <a href="{{ post.url }}" class="tip-title">{{ post.data.title }}</a>
-  {% mainImage post %}
+  <a href="{{ post.url }}" class="tip-image">{% mainImage post %}</a>
+  {# {% mainImage post %} #}
```

Comment:
This looks like it should work. If not you could try `<a href="{{ post.url }}" ><img class="tip-image">{% mainImage post %}</img></a>`.

I would delete your commented out code thou, it is bad manors.

Reply:
Removed Commented Line.